examples/src/main/java/io/dapr/examples/state/http/OrderManager.java

```diff
@@ -42,6 +44,7 @@
 public class OrderManager {
 
   static HttpClient httpClient;
+  static final List<Integer> httpOkStatus = Arrays.asList(200,201);
```

Comment:
HTTP_OK_STATUSES

Reply:
Theoretically this is mutable, so it should be wrapped with Collections.unmodifiableList(). It is simpler to just use OR in the if condition.